### PR TITLE
Focus the Setup Page on the Terraform Installer

### PR DESCRIPTION
**Pull request by blueandgold**
Another step in the process to deprecate the python installer, by focusing the setup page on the terraform installer with a prominent button, and move the python installer's instructions to a separate page.

Would welcome input on wording & structure of the pages.

image

### CODE REVIEW

File: _docs/_latest/setup/install.md [hunk 1]
```diff
@@ -24,111 +24,14 @@ organization Cloud IAM policy.
...
-
-     ```bash
-     MODEL_ID=$(/bin/date -u +%Y%m%dT%H%M%S)
-     forseti inventory create --import_as ${MODEL_ID}
-     forseti model use ${MODEL_ID}
-     forseti scanner run
-     forseti notifier run
-     ```
+If you still need to use the legacy and soon-to-be-deprecated python-based
+installer, it be found [here]({% link _docs/latest/setup/install-with-python-installer.md %}).
```
Comment (hshin-g) [inline _docs/_latest/setup/install.md]: "it be found" --> "it can be found"

File: _docs/_latest/setup/install.md [hunk 2]
```diff
@@ -24,111 +24,14 @@ organization Cloud IAM policy.
 
 ## Setting up Forseti Security
 
-The installer automatically determines setup information, generates a deployment
-template, and creates a Forseti deployment.
+Users can install Forseti using Terraform by clicking the button below.
 
-### Terraform installation
+[![install forseti using cloud shell tutorial]({{site.url}}/images/docs/setup/install_button_using_cloud_shell_tutorial.png)](http://www.google.com)
```
Comment (hshin-g) [inline _docs/_latest/setup/install.md]: Link should be: https://github.com/forseti-security/terraform-google-forseti/tree/master/examples/install_simple

**Comment by morgante** [comment]
@hshin-g Actually, we should probably link directly to the Cloud Shell tutorial like this:

```
[![Open in Cloud Shell](https://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fforseti-security%2Fterraform-google-forseti.git&cloudshell_git_branch=master&cloudshell_working_dir=examples/install_simple&cloudshell_image=gcr.io%2Fgraphite-cloud-shell-images%2Fterraform%3Alatest&cloudshell_tutorial=.%2Ftutorial.md)
```

**Comment by hshin-g** [comment]
Morgante's suggestion works for me!

File: _docs/_latest/setup/install.md (hunk 1 above)
Comment (blueandgold) [reply _docs/_latest/setup/install.md]: Done!

File: _docs/_latest/setup/install.md (hunk 2 above)
Comment (blueandgold) [reply _docs/_latest/setup/install.md]: Make the changes that morgante suggested.

**Pull request merged**